modules/javafx.controls/src/test/java/test/javafx/scene/control/PropertySizeTest.java

```diff
@@ -23,7 +23,7 @@
  * questions.
  */
 
-package test.javafx.css;
+package test.javafx.scene.control;
 
```

Comment:
ahh .. you moved the test into an existing package - that's fine with me if you are certain that we don't want a dedicated package for css? Locally, I create `test.javafx.scene.control.css` and moved the tests there. But really doesn't matter much :)

Reply:
Package `test.javafx.scene.control.css` sounds good to me too. It is changed now.